oap-server/banyandb-java-client/src/test/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClientWriteTest.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.banyandb.v1.client;
+
+import io.grpc.ManagedChannel;
+import io.grpc.inprocess.InProcessChannelBuilder;
+import io.grpc.inprocess.InProcessServerBuilder;
+import io.grpc.stub.StreamObserver;
+import io.grpc.testing.GrpcCleanupRule;
+import io.grpc.util.MutableHandlerRegistry;
+import org.apache.skywalking.banyandb.v1.Banyandb;
+import org.apache.skywalking.banyandb.v1.trace.BanyandbTrace;
+import org.apache.skywalking.banyandb.v1.trace.TraceServiceGrpc;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+public class BanyanDBClientWriteTest {
+    @Rule
+    public final GrpcCleanupRule grpcCleanup = new GrpcCleanupRule();
+
+    private final MutableHandlerRegistry serviceRegistry = new MutableHandlerRegistry();
+
+    private BanyanDBClient client;
+    private TraceBulkWriteProcessor traceBulkWriteProcessor;
+
+    @Before
+    public void setUp() throws IOException {
+        client = new BanyanDBClient("127.0.0.1", 17912, "default");
+
+        String serverName = InProcessServerBuilder.generateName();
+
+        grpcCleanup.register(InProcessServerBuilder
+                .forName(serverName).fallbackHandlerRegistry(serviceRegistry).directExecutor().build().start());
+
+        ManagedChannel channel = grpcCleanup.register(
+                InProcessChannelBuilder.forName(serverName).directExecutor().build());
```

Comment:
After this build, you could use `getPort` for client. Don't use static port, you could face port conflict potentially.

Reply:
`Server.getPort()` is used now. But since it is an inProcess server, it just return `-1` as the port.

Reply:
Yes, no port used makes sense.